client/src/legacy/ElementEditor/entwine.js

```diff
@@ -64,12 +82,20 @@ jQuery.entwine('ss', ($) => {
       root.render(<ElementEditorComponent {...props} />);
     },
 
+    onmatch() {
+      this.render();
+    },
```

Comment:
It doesn't look like `render()` is being called at any other time? Is that happening in one of the other PRs?

Reply:
Removed